kalite/coachreports/static/js/coachreports/student_progress/hbtemplates/playlist-progress-container.handlebars

```diff
@@ -1,28 +1,44 @@
 <div class="well">
     <div class="row">
         <div class="col-xs-12">
-            <h3 class="playlist-title center-block">
+            <h2 class="h3 playlist-title center-block">
                 <a href="{{this.url}}">{{title}}</a>
-            </h3>
+            </h2>
+            <div class="sr-only">{{_ "In this topic " }}</div>
         </div>
     </div>
     {{!-- Videos --}}
     <div class="row progress-block">
         <div class="col-xs-2">
             <div class="progress-indicator progress-indicator-lg center-block {{vid_status}}">
-                <div class="sidebar-icon icon-Video icon-label icon-label-lg"><span class="progress-indicator content-counter label label-primary">{{#if n_pl_videos}}{{n_pl_videos}}{{/if}}</span></div>
+                <span class="sr-only">{{_ "there are " }}</span>
+                <div class="sidebar-icon icon-Video icon-label icon-label-lg"><span class="progress-indicator content-counter label label-primary"><div class="sr-only">{{#unless n_pl_videos}}no{{/unless}}</div>{{#if n_pl_videos}}{{n_pl_videos}}{{/if}}</span></div>
+                <span class="sr-only">{{_ " videos." }}</span>
             </div>
         </div>
         <div class="col-xs-10">
             <div class="progress playlist-progress-bar">
+                {{#if n_pl_videos}}
+                    {{#unless vid_pct_started}}
+                        <span class="sr-only">{{_ "You haven't started to watch videos! " }}</span>
+                    {{/unless}}
+                {{/if}}
                 {{#if vid_pct_complete}}
                     <div class="progress-bar progress-bar-success" title="% Completed" style="width: {{vid_pct_complete}}%">
+                        <span class="sr-only">{{_ "Of those you have completed " }}</span>
                         <span>{{vid_pct_complete}}%</span>
                     </div>
                 {{/if}}
                 {{#if vid_pct_started}}
                     <div class="progress-bar inprogress" title="% In-Progress" style="width: {{vid_pct_started}}%">
+                    {{#ifcond vid_pct_started "<" 100}}
+                        <span class="sr-only">{{_ "You are still working on " }}</span>
                         <span>{{vid_pct_started}}%</span>
+                    {{/ifcond}}
+                    {{#ifcond vid_pct_started "===" 100}}
+                        <span class="sr-only">{{_ "You are still working on all of them." }}</span>
+                        <span aria-hidden="true" role="presentation">{{vid_pct_started}}%</span>
+                    {{/ifcond}}
                     </div>
```

Comment:
No, makes sense. I feel like reviewing this code would be easier with a screen reader!